src/commands/DisableExtensionCommand.ts

```
@@ -0,0 +1,129 @@
+ * ```typescript
+ * const command = new DisableExtensionCommand(configService, logger, accessibilityService);
+ * await command.execute();
+ * ```
+ */
+
+import type { ILogger } from '../di';
+import type { IAccessibilityService } from '../di/interfaces/IAccessibilityService';
+import type { IConfigurationService } from '../di/interfaces/IConfigurationService';
+
+import { BaseCommandHandler  } from './BaseCommandHandler';
+import type {CommandResult} from './BaseCommandHandler';
```

ESLint

[error] './BaseCommandHandler' import is duplicated.

(no-duplicate-imports)